src/constants/account.ts

```diff
@@ -51,3 +51,8 @@ export type Hdkey = {
 };
 
 export const AMOUNT_RESERVED_FOR_GAS_USDC = 0.1;
+
+/**
+ * @description The number of parentSubaccounts: 0 - 127, 128 is the first childSubaccount
+ */
+export const NUM_PARENT_SUBACCOUNTS = 128;
```

Comment:
is this something abacus can specific the position instead of hardcoding 128 on client?

Reply:
do you mean is this something we can export directly from Abacus?

Reply:
yep